### ISSUE TITLE
Allow user to increase gas margin above average gas price

### ISSUE THREAD

**Issue by kauri-hero** [MASQ-Project/Node#505]
Currently the `gas-price` parameter won't be used after [GH-744](https://github.com/MASQ-Project/MASQ-Node-issues/issues/744) is merged in.

To enhance reliable transaction finality, design a new config parameter that allows the user choose to increase the gas above the chain-returned average gas price, so it processes with higher gas cost.

This would likely be blocked by [GH-711 payment adjuster](https://github.com/MASQ-Project/MASQ-Node-issues/issues/711)

**Comment by kauri-hero** [comment]
duplicate of https://github.com/MASQ-Project/MASQ-Node-issues/issues/792
